extensions/vertx-http/deployment/pom.xml

```diff
@@ -12,6 +12,12 @@
     <artifactId>quarkus-vertx-http-deployment</artifactId>
     <name>Quarkus - Vert.x - HTTP - Deployment</name>
 
+    <properties>
+        <bootstrap.version>4.6.0</bootstrap.version>
+        <fontawesome.version>5.15.2</fontawesome.version>
+        <jquery.version>3.5.1</jquery.version>
```

Comment:
Would it be possible to include them in the build-parent instead (not the BOM!)?

Reply:
Sure.

Reply:
I think it would also be nice to have `webjars` in the property name, to indicate this is not a regular Java library